search_objects/src/search_objects.py

```diff
@@ -13,13 +13,22 @@ def search_objects_data(search, limit):
     }
 
 def handler(event, context):
-    search = event ['queryStringParameters']['search']
-    limit = int(10)
-    items = search_objects_data(search,limit)
-    return {
-        'statusCode': 200,
-        'headers': {
-            'Content-Type': 'application/json'
-        },
-        'body': json.dumps(items)
-    }
+    try:
+        search = event ['queryStringParameters']['search']
```

Comment:
a validation with if should be enough. See PR 78 comment.

Reply:
tracked in issue 71